### PR TITLE
Fixed: Update from version in logs

### LINKED ISSUE #873: Pull Radarr commit 'Fixed: Update from version in logs'
Pull https://github.com/Radarr/Radarr/commit/bc3e3714b97dec19091e45324d9f2b550f1dbbd5

```
git cherry-pick -ex bc3e3714b97dec19091e45324d9f2b550f1dbbd5
```

This commit has 1 conflict(s)

[AB#2744](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/2744)

### LINKED ISSUE #877: Pull Radarr commit 'Fixed: Updater version number logging'
Pull https://github.com/Radarr/Radarr/commit/b34f4fde1bbe611bb5c886890e3be7cec2b96e6b

```
git cherry-pick -ex b34f4fde1bbe611bb5c886890e3be7cec2b96e6b
```

This commit has 1 conflict(s)

You cannot ask me to `close all`, `open pr here` or `open pr everywhere`

[AB#2761](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/2761)

### PR DESCRIPTION
**Pull request by bakerboy448**
Closes #873

(cherry picked from commit bc3e3714b97dec19091e45324d9f2b550f1dbbd5)

#### Database Migration
NO

#### Description
A few sentences describing the overall goals of the pull request's commits.

#### Todos
- [ ] Tests
- [ ] Translation Keys (./src/NzbDrone.Core/Localization/Core/en.json)
- [ ] [Wiki Updates](https://wiki.servarr.com)

#### Issues Fixed or Closed by this PR

* Fixes #877
* Fixes #873

### CODE REVIEW

File: src/NzbDrone.Update/UpdateEngine/DetectExistingVersion.cs [hunk 1]
```diff
@@ -22,7 +24,7 @@ public string GetExistingVersion(string targetFolder)
         {
             try
             {
-                var targetExecutable = Path.Combine(targetFolder, "Prowlarr.exe");
+                var targetExecutable = Path.Combine(targetFolder, BuildInfo.AppName.ProcessNameToExe());
```
Comment (Qstick) [inline src/NzbDrone.Update/UpdateEngine/DetectExistingVersion.cs:27]: Prowlarr.dll or BuildInfo.AppName + ".dll"

**Review by Qstick — CHANGES_REQUESTED** [review]
Bring it up to date

**Comment by Qstick** [comment]
Original Author is dropped

**Comment by bakerboy448** [comment]
> Original Author is dropped

fixed

**Comment by Qstick** [comment]
> > Original Author is dropped
> 
> fixed

Sure?

**Comment by Qstick** [comment]
No fixy
image

**Comment by bakerboy448** [comment]
now fixy

**Review by Qstick — APPROVED** [review]

**Pull request merged**